package.json

```diff
@@ -23,22 +23,22 @@
     "!src/*.spec.js"
   ],
   "devDependencies": {
-    "@babel/core": "^7.18.6",
-    "@babel/preset-env": "^7.18.6",
-    "@babel/preset-typescript": "^7.18.6",
-    "@types/jest": "^28.1.4",
-    "@types/node": "^16.11.43",
-    "@types/puppeteer": "^5.4.6",
+    "@babel/core": "^7.21.0",
+    "@babel/preset-env": "^7.20.2",
+    "@babel/preset-typescript": "^7.21.0",
+    "@types/jest": "^29.4.0",
+    "@types/node": "^18.14.6",
+    "@types/puppeteer": "^5.4.7",
```

Comment:
Delete `@types/puppeteer`, because it's deprecated, see https://www.npmjs.com/package/@types/puppeteer